vehicle/as/src/pacmod_interface/pacmod_interface.cpp

```diff
@@ -226,24 +241,24 @@ void PacmodInterface::callbackPacmodRpt(
 
   /* publish vehicle status control_mode */
   {
-    autoware_vehicle_msgs::msg::ControlMode control_mode_msg;
-    control_mode_msg.header = header;
+    autoware_auto_vehicle_msgs::msg::ControlModeReport control_mode_msg;
+    control_mode_msg.stamp = header.stamp;
 
     if (!global_rpt->enabled) {
-      control_mode_msg.data = autoware_vehicle_msgs::msg::ControlMode::MANUAL;
+      control_mode_msg.mode = autoware_auto_vehicle_msgs::msg::ControlModeReport::MANUAL;
     } else if (is_pacmod_enabled_) {
-      control_mode_msg.data = autoware_vehicle_msgs::msg::ControlMode::AUTO;
+      control_mode_msg.mode = autoware_auto_vehicle_msgs::msg::ControlModeReport::AUTONOMOUS;
     } else {
-      bool is_pedal_enable = (accel_rpt_ptr_->enabled && brake_rpt_ptr_->enabled);
-      bool is_steer_enable = steer_wheel_rpt_ptr_->enabled;
+      const bool is_pedal_enable = (accel_rpt_ptr_->enabled && brake_rpt_ptr_->enabled);
+      const bool is_steer_enable = steer_wheel_rpt_ptr_->enabled;
       if (is_pedal_enable) {
-        control_mode_msg.data = autoware_vehicle_msgs::msg::ControlMode::AUTO_PEDAL_ONLY;
+        control_mode_msg.mode = autoware_auto_vehicle_msgs::msg::ControlModeReport::AUTONOMOUS;
       } else if (is_steer_enable) {
-        control_mode_msg.data = autoware_vehicle_msgs::msg::ControlMode::AUTO_STEER_ONLY;
+        control_mode_msg.mode = autoware_auto_vehicle_msgs::msg::ControlModeReport::AUTONOMOUS;
```

Comment:
AUTO_PEDAL_ONLY/STEER_ONLY are removed in ControlModeCommand/Report in .auto. Is it ok to return Autonomous from pacmod to autoware?
@tkimura4

Reply:
> AUTO_PEDAL_ONLY/STEER_ONLY are removed

Then, Autoware does not support these mode, so these mode should not be considered AUTO-MODE.
I think following is better.

if (global_rpt->enabled && is_pacmod_enabled_) {
      control_mode_msg.mode = autoware_auto_vehicle_msgs::msg::ControlModeReport::AUTONOMOUS;
    } else {
      control_mode_msg.mode = autoware_auto_vehicle_msgs::msg::ControlModeReport::MANUAL;
    }

cc. @TakaHoribe

Reply:
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/543/commits/c36fc4f3cf536d83a74a0bcfcf9a47b489dae139
DONE